clients/src/main/resources/common/message/OffsetFetchRequest.json

```diff
@@ -39,6 +41,8 @@
         "about": "The topic name."},
       { "name": "PartitionIndexes", "type": "[]int32", "versions": "0+",
         "about": "The partition indexes we would like to fetch offsets for." }
-    ]}
+    ]},
+    {"name": "RequireStable", "type": "bool", "versions": "7+", "default": "false",
+     "about": "The flag indicating whether we should retry for pending transaction commit completion to get stable offset"}
```

Comment:
nit: .. whether broker should hold on returning unstable offsets but set a retriable error code for the partition.